app/models/purl.rb

```diff
@@ -20,6 +20,29 @@ class Purl < ActiveRecord::Base
   }
 
   ##
+  # Return true targets with always values only if the object is not deleted in
+  # purl mount
+  # @return [Array]
+  def computed_true_targets
```

Comment:
is there a case where `true_targets` should return the actual true targets of a deleted PURL? in other words, can you just put this logic into `true_targets`?